docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -890,36 +890,217 @@ Combining hashes/dictionaries
 
 .. versionadded:: 2.0
 
-The `combine` filter allows hashes to be merged. For example, the
-following would override keys in one hash::
+The `combine` filter allows hashes to be merged.
+For example, the following would override keys in one hash::
 
     {{ {'a':1, 'b':2} | combine({'b':3}) }}
 
 The resulting hash would be::
 
     {'a':1, 'b':3}
 
-The filter also accepts an optional `recursive=True` parameter to not
-only override keys in the first hash, but also recurse into nested
-hashes and merge their keys too
+The filter can also take multiple arguments to merge::
+
+    {{ a | combine(b, c, d) }}
+    {{ [a, b, c, d] | combine }}
+
+In this case, keys in `d` would override those in `c`, which would
```

Comment:
```suggestion
In this case, keys in ``d`` would override those in ``c``, which would
```
This is Restructured Text, so use double backticks for code formatting.

Reply:
done (plus all the other ` in the combine section)